orderbook/src/database/events.rs

```diff
@@ -15,12 +15,12 @@ pub struct EventIndex {
 
 #[derive(Debug)]
 pub enum Event {
-    Trade(Trade),
+    DbTrade(DbTrade),
     Invalidation(Invalidation),
 }
 
 #[derive(Debug, Default)]
-pub struct Trade {
+pub struct DbTrade {
```

Comment:
Nit: I feel this rename is unnecessary because we are  already in the database module. If the problem is that we already have `Trade` in `model` and you want to use both structs then you can solve that by importing with a different name like `use database::events::Trade as DbTrade`.

Reply:
Yea cool, I can fix this. Great suggestion.